src/index.html

```diff
@@ -7,8 +7,41 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#">
+        <div class="img"></div>
+      </a>
+      <nav class="nav">
+        <ul class="nav__lists">
+          <li class="nav__list_link">
+            <a class="is-active link" href="#">Apple</a>
+          </li>
```

Comment:
Let's fix it everywhere
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a class="is-active nav__link" href="#">Apple</a>
          </li>
          
```